intranet/templates/eighth/admin/schedule_activity.html

```diff
@@ -18,6 +18,7 @@
 
     <script>
     $(function() {
+        var activityIsSticky = "{{ activity.sticky }}";
```

Comment:
This won't do what you want.

1. You need to move this out of the initialization function. Currently, it's scoped locally to that function, and the other code can't see it.
2. You're assigning this variable the *string* `"True"` or `"False"`; your comparison in `static/js/eighth/schedule.js` will always fail. Instead, try taking a look at the setting of `window.isAdmin` in `templates/signage/pages/bus.html`. That's a working of setting a Javascript boolean variable in a Django template.

Reply:
Yeah I was pretty sure this was wrong, not super familiar with Django templates.